docs/docs/kds-rest-api.md

```diff
@@ -131,15 +131,14 @@ cannot be accessed by the altered key specification.
 }
 ```
 
-| Field | Description |
-| ----- | ----------- |
-| kdsAttestationReport | The `EnclaveInstanceInfo` in Base64 of the KDS enclave. The application enclave should validate this report before trusting the private key returned by the KDS enclave. |
-| encryptedPrivateKey | A Base64 field that contains the private key packaged as a Mail object encrypted using the application enclave key. The application enclave will decrypt the Mail object to extract the private key. The envelope in the Mail contains the name, masterKeyType, and policyConstraint parameters of the KDS request. These parameters must be checked against the original. See below how to deserialize the envelope.|
+| Field                 | Description                                                                                                                                                                                                                                                                                                                                                                                                           |
+|-----------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| kdsAttestationReport  | The `EnclaveInstanceInfo` in Base64 of the KDS enclave. The application enclave should validate this report before trusting the private key returned by the KDS enclave.                                                                                                                                                                                                                                              |
+| encryptedPrivateKey   | A Base64 field that contains the private key packaged as a Mail object encrypted using the application enclave key. The application enclave decrypts the Mail object to extract the private key. The envelope in the Mail contains the name, masterKeyType, and policyConstraint parameters of the KDS request. These parameters must be checked against the original. See below for how to deserialize the envelope. |
```

Comment:
Mail -> Mail object

Also "the original" is a confusing way to put it, what "original"? Change the second to last sentence to something along these lines:

These parameters must be checked against the parameters in the original request to verify that the key was derived correctly.